typescript/web/src/components/members/new-member-modal.tsx

```diff
@@ -1,35 +1,39 @@
-import { useState, ChangeEvent } from "react";
 import {
-  Flex,
   Alert,
-  AlertIcon,
   AlertDescription,
-  VStack,
-  Text,
-  Textarea,
+  AlertIcon,
+  Button,
+  Flex,
+  Heading,
+  Link,
   Modal,
-  ModalOverlay,
-  ModalContent,
+  ModalBody,
   ModalCloseButton,
+  ModalContent,
   ModalFooter,
-  Button,
-  Heading,
   ModalHeader,
-  ModalBody,
+  ModalOverlay,
+  Text,
+  Textarea,
   useColorModeValue as mode,
-  Link,
   useToast,
+  VStack,
 } from "@chakra-ui/react";
+import { InvitationResult, MembershipRole } from "@labelflow/graphql-types";
+import { isEmpty } from "lodash/fp";
 import { useRouter } from "next/router";
-import { MembershipRole, InvitationResult } from "@labelflow/graphql-types";
-
+import { ChangeEvent, useState } from "react";
+import { validateEmail } from "../../utils/validate-email";
 import { RoleSelection } from "./role-selection";
 import { InviteMember } from "./types";
-import { validateEmail } from "../../utils/validate-email";
```

Comment:
@nicOwlas Apart from the new `isEmpty` import from `lodash/fp`, the changes above are only caused by the "Organize imports" feature of VS Code which lets me gain hundreds of cumulated hours reorganizing imports manually by doing it automatically.

_It might be nice to enable it by default in the future (and ignore it in the rare places where the import order actually matters) so that the whole team could gain some dev time too._

Reply:
Thx for the review and the changes. Let's expedite those small PR and focus on the more important features. Understood for the `Organize imports`. I've enabled it but I cannot find the option to avoid putting grouped imports on the same line.